src/mechanics/timeIntegration/NewmarkDirect.h

```diff
@@ -30,26 +30,11 @@ class NewmarkDirect : public NewmarkBase
     //! @brief constructor
     NewmarkDirect(StructureBase* rStructure);
 
-    void SetMinLineSearchStep(double rMinLineSearchStep)
-    {
-    	mMinLineSearchStep = rMinLineSearchStep;
-    }
-
-    double GetMinLineSearchStep()const
-    {
-    	return mMinLineSearchStep;
-    }
-
     void SetPerformLineSearch(bool rPerformLineSearch)
     {
         mPerformLineSearch = rPerformLineSearch;
     }
 
-    bool GetPerformLineSearch() const
```

Comment:
Well, in that case we should probably implement a method that _streams_ all necessary information and not just a function that returns a copy or reference to that variable.